teuthology/task/install.py

```diff
@@ -702,28 +702,45 @@ def rh_install(ctx, config):
     :param ctx: the argparse.Namespace object
     :param config: the config dict
     """
+    yaml_path = None
+    # Look for rh specific packages in <suite_path>/rh/downstream.yaml
+    if 'suite_path' in ctx.config:
+        ds_yaml = os.path.join(
+            ctx.config['suite_path'],
+            'rh',
+            'downstream.yaml',
+        )
+        if os.path.exists(ds_yaml):
+            yaml_path = ds_yaml
+    # default to user home dir if one exists
+    default_yaml = os.path.expanduser('~/downstream.yaml')
+    if os.path.exists(default_yaml):
+        yaml_path = default_yaml
+    rh_ds_yaml = yaml.safe_load(open(yaml_path))
```

Comment:
This variable name is misleading. Once you parse yaml, it's not yaml anymore but an object. Perhaps ``downstream_config`` or something?